masschallenge/impact-api
# [AC-6851] People Directory - Use people profile IDs not Team member IDs

**Changes Introduced in [AC-6851](https://masschallenge.atlassian.net/browse/AC-6851):**
-

**Test:**

_On development or Staging_
- Visit people directory http://localhost:1234/people or https://staging.masschallenge.org/people
- Search for `des pieri`
- Notice that clicking the user, the people directory just reloads

_checkout this branch or Test5_
- Visit people directory http://localhost:1234/people or https://test5.masschallenge.org/people
- Search for `des pieri`
- Notice that this time around, clicking the user shows the profile

Sibling PR: https://github.com/masschallenge/front-end/pull/171
